### PR TITLE
Update usage to the new yml syntax

### PR DESCRIPTION
**Pull request by smorimoto**
This Pull Request has the same purpose as https://github.com/actions/npm/pull/42.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,47 +6,59 @@ Here are a few use cases for this action:
...
-}
-
-action "Apply Triage Label" {
-  uses = "actions/github@v1.0.0"
-  args = "label triage --action=opened" # Only when issues are opened!
-  secrets = ["GITHUB_TOKEN"]
-}
+```yml
+name: Triage
+on: [issues]
```
Comment (andymckay) [inline README.md]: We could do this by:
```
on:
  issues:
    types: [opened]
```

File: README.md [hunk 2]
```diff
@@ -6,47 +6,59 @@ Here are a few use cases for this action:
...
+    name: Apply Triage Label
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Apply Triage Label
+        uses: actions/github@v1.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          args: label triage --action=opened
```
Comment (andymckay) [inline README.md]: If we do that, we could remove `--action=opened` here I presume?

File: README.md [hunk 3]
```diff
@@ -6,47 +6,59 @@ Here are a few use cases for this action:
...
-}
-
-action "Comment On New Issues" {
-  uses = "actions/github@v1.0.0"
-  args = "comment "Hello World" --action=opened" # Only when issues are opened!
-  secrets = ["GITHUB_TOKEN"]
-}
+```yml
+name: Triage
+on: [issues]
```
Comment (andymckay) [inline README.md]: Repeat on here and the next one.

**Review by andymckay — COMMENTED** [review]
I think with those changes we'll be good to go.

File: README.md (hunk 1 above)
Comment (smorimoto) [reply README.md]: Oh I didn't know that. Cool.

File: README.md (hunk 2 above)
Comment (smorimoto) [reply README.md]: Yes.

File: README.md (hunk 3 above)
Comment (smorimoto) [reply README.md]: 👌

**Comment by smorimoto** [comment]
@andymckay Updated!

**Review by andymckay — APPROVED** [review]
Great thank you @imbsky

**Comment by smorimoto** [comment]
You’re welcome.

**Comment by smorimoto** [comment]
@andymckay I sent similar PR to other repositories. Old documents confuse many people, so we should update them as soon as possible.

**Comment by smorimoto** [comment]
Oh, most other repositories seem to have been deleted or archived.

**Comment by andymckay** [comment]
Thank you for doing that work, but we've found most of the other repositories are just not relevant any more and we agree confuse people - so trying to clean those up too.

**Comment by smorimoto** [comment]
It makes sense!

**Pull request merged**